File: Maths/Series/Lucas_Series/lucas_series.ts
// This is required for console input
var readline = require('readline');
var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Number of terms to be found: ", function(answer) {
    let first = 2;
    let second = 1;
    let result;
    console.log(first);
    console.log(second);
  for (let i = 0; i < +answer - 2; i++) {
    result = first + second;
    console.log(result);
    first = second;
    second = result;
  }
  rl.close();
});

File: Maths/Series/Lucas_Series/Lucas_Series.cpp
#include <iostream>
using namespace std;

int main() {

    int term_1 = 2;
    int term_2 = 1;
    int next_term;
    
    int num_terms;  
    cout << "Number of terms to be found: ";
    cin >> num_terms;

    cout << "The first " << num_terms << " of the Lucas Series are:\n";

    for (int i = 1; i <= num_terms; i++) {
        cout << term_1 << "\t";
        next_term = term_1 + term_2;
        term_1 = term_2;
        term_2 = next_term;
    }
    
    cout << "\n";
}